mixer/pkg/runtime2/routing/monitoring_test.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package routing
+
+import (
+	"testing"
+	"time"
+
+	tpb "istio.io/api/mixer/v1/template"
+	"istio.io/istio/mixer/pkg/runtime2/testing/data"
+)
+
+func TestDestinationCounters_Update(t *testing.T) {
+	// Ensure that the call doesn't crash
+	serviceConfig := data.ServiceConfig
```

Comment:
as above, should we add a panic handler to log and fail here?

Reply:
yeah, this was more of a nit. by capturing it and failing explicitly we can provide more context about how to interpret (or, at least that was my thought).